chain/ethereum/src/network_indexer/convert.rs

```diff
@@ -113,9 +169,101 @@ impl TryIntoEntity for &BlockWithOmmers {
                     .collect::<Vec<_>>()
                     .into(),
             ),
+            ("ommerHash", inner.uncles_hash.into()),
+            (
+                "transactions",
+                (inner
+                    .transactions
+                    .iter()
+                    .map(move |transaction| transaction.hash.to_entity_id())
+                    .collect::<Vec<_>>()
+                    .into()),
+            ),
+            ("logCount", (logs.len() as i32).into()),
+            ("logs", logs.into()),
             ("size", inner.size.into()),
             ("sealFields", inner.seal_fields.clone().into()),
-            ("isOmmer", false.into()),
+        ] as Vec<(_, Value)>))
+    }
+}
+
+impl TryIntoEntity for &LoadedTransaction {
+    fn try_into_entity(self) -> Result<Entity, Error> {
+        let transaction = &self.transaction;
+        let receipt = &self.receipt;
+
+        Ok(Entity::from(vec![
```

Comment:
It could be worth switching to the `object!` macro in a separate commit. We can always do that later though, so I'll leave it up to you.